src/effects/backends/audiounit/audiounitmanifest.mm

```diff
@@ -18,9 +20,18 @@
     setDescription(QString::fromNSString([component typeName]));
     setAuthor(QString::fromNSString([component manufacturerName]));
 
-    // Try instantiating the unit in-process to fetch its properties quickly
+    // Instantiate audio unit (out-of-process) to load parameters
+    AudioUnitManager manager{component};
+
+    const int TIMEOUT_MS = 2000;
+    if (!manager.waitForAudioUnit(TIMEOUT_MS)) {
+        qWarning() << name() << "took more than" << TIMEOUT_MS
+                   << "ms to initialize, skipping manifest initialization "
+                      "for this effect. This means this effect will not "
+                      "display any parameters and likely not be useful!";
+        return;
+    }
```

Comment:
I just had another idea what the issue could be... if we bail out here (due to a timeout), we deinitialize the `AudioUnitManager` instance early, which may cause the async initialization callback to get a dangling (captured) `this` pointer

Reply:
I can reproduce the crash relatively reliably by lowering the timeout, so could indeed be it...

Reply:
Fixed in 10140c9